src/components/kepler-gl.js

```diff
@@ -204,6 +207,11 @@ function KeplerGlFactory(
         clicked
       } = visState;
 
+      const notificationPanelFields = {
+        uiStateActions,
+        uiState
```

Comment:
If you are not using other properties in `uiState`, should just pass in `uiState.notifications` here, making the `notifications -> notificationPanel` logic simple and clear

Reply:
I have been thinking about this, in fact the first version had only notifications. I thought that passing the full uiState would provide more information and context about the current state of the UI. Currently we only use notifications but maybe in the future we could take advantage of other properties to change the notification panel behavior

Reply:
Let's only use notificaitons and removeNotification as action